cmd/drand-cli/public.go

```diff
@@ -107,7 +107,7 @@ func getChainInfo(c *cli.Context) error {
 	var err error
 	chainHash := make([]byte, 0)
 	if c.IsSet(hashInfoNoReq.Name) {
-		if chainHash, err = hex.DecodeString(c.String(hashInfoNoReq.Name)); err != nil {
+		if chainHash, err = hex.DecodeString(c.String(hashInfoReq.Name)); err != nil {
```

Comment:
Wait, this is doing `if c.IsSet(hashInfoNoReq.Name)` and then `hex.DecodeString(c.String(hashInfoReq.Name))`.

This seems terribly wrong.

Reply:
Oops true - they coincidentally have the same Name value though!